hnn_core/network.py

```diff
@@ -581,19 +590,21 @@ def add_poisson_drive(self, name, *, tstart=0, tstop=None, rate_constant,
         drive['type'] = 'poisson'
         drive['n_drive_cells'] = n_drive_cells
         drive['seedcore'] = seedcore
+        drive['conn_seed'] = conn_seed
         drive['dynamics'] = dict(tstart=tstart, tstop=tstop,
                                  rate_constant=rate_constant)
         drive['events'] = list()
 
         self._attach_drive(name, drive, weights_ampa, weights_nmda, location,
                            space_constant, synaptic_delays,
-                           n_drive_cells, cell_specific)
+                           n_drive_cells, cell_specific, probability)
 
     def add_bursty_drive(self, name, *, tstart=0, tstart_std=0, tstop=None,
                          location, burst_rate, burst_std=0, numspikes=2,
                          spike_isi=10, n_drive_cells=1, cell_specific=False,
                          weights_ampa=None, weights_nmda=None,
-                         synaptic_delays=0.1, space_constant=100., seedcore=2):
+                         synaptic_delays=0.1, space_constant=100.,
+                         probability=1.0, seedcore=2, conn_seed=2):
```

Comment:
why not reuse seedcore?

Reply:
Personally I think this is pretty important to decouple the random seeding of the events vs connections. I can certainly imagine a case for wanting to keep the same physical connectivity structure while changing the event times.

Perhaps rename `seedcore` to `event_seed`?

Reply:
okay, yeah we can call it `event_seed`. By the way, we are doing this kind of renaming very liberally right now but in a more stable software it would break user scripts. I think we can continue to do this until 0.2 but start getting into the habit of deprecation warnings after that.

Also, the more I think, the more I like your proposal of making `Drives` into an object like cells so you can just use the network connectivity API.